website/app/index.html

```diff
@@ -4,6 +4,8 @@
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1">
+    <!-- TODO! remember to remove this once we're ready to launch -->
+    <meta name="robots" content="noindex">
```

Comment:
```suggestion
    <meta name="robots" content="noindex, nofollow">
```

I usually use these both- it's saying "don't index the page and don't crawl any links on the page"